blueos_repository/registry.py

```diff
@@ -0,0 +1,81 @@
+from typing import Any, Dict, List, Optional
+
+import aiohttp
+
+
```

Comment:
It would be really nice to have the documentation for the docker API in the docstring of the class.

Reply:
done